lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
@andy-armstrong What they probably should use is:
```
<%= HtmlUtils.HTML(myHtml) %>
```
The problem, if we think it is one, is that the HTML() call would open up a vulnerability if they are wrong, while escape() would not.  I think that is why Dan had proposed using `safe` in some cases.  Do we want to introduce a different function name that would be a synonym to escape, but used in certain context?